scalafmt-core/shared/src/main/scala/org/scalafmt/internal/Router.scala

```diff
@@ -36,6 +36,8 @@ import org.scalafmt.util.SomeInterpolate
 import org.scalafmt.util.TokenOps
 import org.scalafmt.util.TreeOps
 import org.scalafmt.util.Trivia
+
+import scala.meta.Decl.Def
```

Comment:
By convention we use `Decl.Def` instead of `import Decl.Def; Def`. This convention prevents conflicts since there is another `Defn.Def`